### PR TITLE
Testing adding iOS matrix.

### PR DESCRIPTION
**Pull request by seivan**
I'm not sure `GLFW3` would work for iOS, but lets see what Travis says?

### CODE REVIEW

**Comment by kvark** [comment]
I don't see our build failure in CI log

**Comment by seivan** [comment]
@kvark Just to check with you, would `cargo test` suffice? Or does it need to run compute triangle?

**Comment by seivan** [comment]
I wanted to see if `cargo test` works for `aarch64`, restarting the other platforms now.

File: .travis.yml [hunk 1]
```diff
@@ -32,10 +32,10 @@ matrix:
       compiler: clang
 
     # iPhoneOS 64bit
-    #- env: TARGET=aarch64-apple-ios
-    #  os: osx
-    #  osx_image: xcode9.4
-    #  rust: nightly
+    - env: TARGET=aarch64-apple-ios
+      os: osx
+      osx_image: xcode11
```
Comment (grovesNL) [inline .travis.yml]: nit: it might be useful to keep the Xcode version low for backwards compatibility

**Comment by kvark** [comment]
bors try

**Comment by kvark** [comment]
Could you rebase/squash please?

File: .travis.yml (hunk 1 above)
Comment (seivan) [reply .travis.yml]: @grovesNL To clarify you mean adding both or **just** testing older ones?

**Comment by kvark** [comment]
@seivan please squash and rebase

**Comment by seivan** [comment]
Sorry, I must missed have missed this. Will get on it tomorrow! 👌🏼

**Comment by seivan** [comment]
@kvark Sorry for the delay, done.

**Comment by seivan** [comment]
> It seems like the two Travis statuses happened because this branch is opened from gfx-rs instead of a fork

Do you want forks in the future instead of a new branch on the same repo?

**Comment by seivan** [comment]
> It seems like the two Travis statuses happened because this branch is opened from gfx-rs instead of a fork

@grovesNL 
Do you want forks in the future instead of a new branch on the same repo?

**Comment by grovesNL** [comment]
@seivan it's not too important, but I think it's slightly better use branches from a personal fork (mostly to reduce the number of active branches in this repository)

**Pull request merged**